app/graphql/mutations/account/set_site_permissions.rb

```diff
@@ -0,0 +1,21 @@
+class Mutations::Account::SetSitePermissions < Mutations::Base
+  argument :input, Types::Input::Account::SetSitePermissions,
+    required: true
+
+  field :account, Types::Account, null: false
+
+  def authorized?(input:)
+    super(input.account, :set_site_permissions?)
+  end
+
+  def resolve(input:)
+    permissions = input.permissions.map(&:to_sym)
+
+    res = Accounts::SetSitePermissions.call(
+      user: input.account,
+      permissions: permissions
+    )
```

Comment:
What type of errors can happen here and should we account for them?

Reply:
`UnknownPermissions` could happen due to a programming error but otherwise it won't happen, and the only other ones are `RecordNotFound` or `RecordInvalid` which I believe are accounted for automatically